README.md

```diff
@@ -191,19 +191,19 @@ idseq-cli-v2 can be configured with environment variables or files. By default c
 - Distributed as a single binary for easier installation that doesn't rely on dependencies on the user's machine
 - Shell completion support
 - Structured with commands and subcommands to make room for future functionality
-- Log in in the web via your IDSeq account instead of using a static token
+- Log in in the web via your CZ ID account instead of using a static token
 - Critical bugfixes
 - Uploads without the need for user prompts
 - Supports configuration files and environment variable configuration instead of relying solely on flags
 
 ## Contributing
 
-This project is not seeking contributions at this time. It is tighly coupled to the IDSeq Web App, it's features, it's APIs, and it's development goals. Please feel free to raise issues for feature requests or bugs.
+This project is not seeking contributions at this time. It is tighly coupled to the CZ ID Web App, it's features, it's APIs, and it's development goals. Please feel free to raise issues for feature requests or bugs.
 
 This project adheres to the Contributor Covenant [code of conduct](https://www.contributor-covenant.org/). By participating, you are expected to uphold this code. Please report unacceptable behavior to opensource@chanzuckerberg.com.
 
 ## Reporting Security Issues
 
-Please note: If you believe you have found a security issue, please responsibly disclose by contacting us at security@idseq.net.
+Please note: If you believe you have found a security issue, please responsibly disclose by contacting us at security@czid.org.
```

Comment:
we haven't updated the emails yet so we should change this back for now

Reply:
good catch!